library/src/test/java/bio/terra/pfb/PfbReaderTest.java

```diff
@@ -0,0 +1,57 @@
+package bio.terra.pfb;
+
+import bio.terra.pfb.utils.CompareOutputUtils;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+class PfbReaderTest {
+
+  @Test
+  void showSchemaTest() throws IOException {
+    var listOfTestFiles = List.of("minimal_data", "minimal_schema", "test", "kf");
```

Comment:
Bit of a nit: looks like all the test files are the same between tests, could we pull them out into a class variable?

Reply:
Yes, can do!